zulipterminal/model.py

```diff
@@ -1215,6 +1215,9 @@ def set_stream_message_retention_days(
     ) -> None:
         self._get_stream_from_id(stream_id)["message_retention_days"] = value
 
+    def is_stream_invite_only(self, stream_id: int) -> bool:
+        return self._get_stream_from_id(stream_id)["invite_only"]
```

Comment:
Similarly to is_web_public, we already have a method for this, which you've updated to use this accessor.

There is no use of this data outside of the model in any case, other than through stream_access_type(), so there is no motivation for introducing this accessor.